_data/setup-scala.yml

```diff
@@ -2,5 +2,5 @@ linux-x86-64: curl -fL https://github.com/coursier/coursier/releases/latest/down
 linux-arm64: curl -fL https://github.com/VirtusLab/coursier-m1/releases/latest/download/cs-aarch64-pc-linux.gz | gzip -d > cs && chmod +x cs && ./cs setup
 macOS-x86-64: curl -fL https://github.com/coursier/coursier/releases/latest/download/cs-x86_64-apple-darwin.gz | gzip -d > cs && chmod +x cs && (xattr -d com.apple.quarantine cs || true) && ./cs setup
 macOS-arm64: curl -fL https://github.com/VirtusLab/coursier-m1/releases/latest/download/cs-aarch64-apple-darwin.gz | gzip -d > cs && chmod +x cs && (xattr -d com.apple.quarantine cs || true) && ./cs setup
-macOS-brew: brew install coursier/formulas/coursier && cs setup
+macOS-brew: brew install coursier && coursier setup
```

Comment:
Why the change from `cs` to `coursier`? I've never been clear on the difference. I think I usually see `cs`?

Reply:
Because for me, `brew install coursier` only installs `coursier`. It's then `coursier setup` that installs other things such as `cs`:

```
➜  ~ brew install coursier
...
➜  ~ which coursier   
/usr/local/bin/coursier
➜  ~ which cs
cs not found
➜  ~ coursier setup
...
Checking if the standard Scala applications are installed
  Installed ammonite
  Installed cs
  Installed coursier
  Installed scala
  Installed scalac
  Installed scala-cli
  Installed sbt
  Installed sbtn
```

Reply:
There's a difference between the homebrew-core formula and couriers own tap - core doesn't include cs but the tap does

Reply:
but seeing as the tap doesn't have an option for apple silicon this is better

Reply:
@SethTisue the difference is that coursier is a jar-based launcher rather than graalvm native image

Reply:
okay, I mean at some level this is all deeply confusing and clear as mud, but it sounds like this is mergeable :shrug: